dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -133,18 +76,33 @@ protected AlluxioFuse() {
    * @throws ParseException
    */
   public void start(String[] args) throws ParseException {
-    CommandLine cli = PARSER.parse(OPTIONS, args);
-
-    if (cli.hasOption(HELP_OPTION_NAME)) {
-      final HelpFormatter fmt = new HelpFormatter();
-      fmt.printHelp(AlluxioFuse.class.getName(), OPTIONS);
+    FuseCliOptions fuseCliOptions = new FuseCliOptions();
+    JCommander jCommander = JCommander.newBuilder()
+        .addObject(fuseCliOptions)
+        .build();
+    jCommander.parse(args);
+    if (fuseCliOptions.getHelp().orElse(false)) {
+      jCommander.usage();
       return;
     }
 
     LOG.info("Alluxio version: {}-{}", RuntimeConstants.VERSION, ProjectConstants.REVISION);
-    setConfigurationFromInput(cli, Configuration.modifiableGlobal());
+    InstancedConfiguration configFromCli = parseCliOptionsAsConfig(fuseCliOptions);
+    { // !!! we are fiddling with mutable global states here !!!
+      // use an explicit scope to limit the visibility of `globalConf`
+      InstancedConfiguration globalConf = Configuration.modifiableGlobal();
+      globalConf.merge(configFromCli.getProperties());
+      Source metadataCacheSizeSource =
+          globalConf.getSource(PropertyKey.USER_METADATA_CACHE_MAX_SIZE);
+      if (metadataCacheSizeSource == Source.DEFAULT
+          || metadataCacheSizeSource == Source.CLUSTER_DEFAULT) {
+        globalConf.set(PropertyKey.USER_METADATA_CACHE_MAX_SIZE, 20000, Source.RUNTIME);
+        LOG.info("Set default metadata cache size to 20,000 entries "
+            + "with around 40MB memory consumption for FUSE");
+      }
```

Comment:
I don't understand why it is hard-coded to overwrite a default configuration.
If such, we can change the default value for this config key.

I know this is from the original code. But please talk to @LuQQiu to fix this.